src/main/java/org/apache/sling/jcr/contentloader/PathEntry.java

```diff
@@ -100,6 +108,24 @@
      */
     public static final String IGNORE_CONTENT_READERS_DIRECTIVE = "ignoreImportProviders";
 
+    /** Used in https://github.com/apache/sling-maven-plugin/blob/d67d5c1900782c68c1bfcbdc499c02cf31224916/sling-maven-plugin/src/main/java/org/apache/sling/maven/bundlesupport/fsresource/SlingInitialContentMounter.java#L97 */
+    private static final String MAVEN_MOUNT_DIRECTIVE = "maven:mount";
+
+    /** All directive names which are valid for header Sling-Initial-Content */
+    public Set<String> VALID_DIRECTIVES = new HashSet<>(Arrays.asList(
+        OVERWRITE_DIRECTIVE,
+        OVERWRITE_PROPERTIES_DIRECTIVE,
+        MERGE_PROPERTIES_DIRECTIVE,
+        MERGE_NODES_DIRECTIVE,
+        UNINSTALL_DIRECTIVE,
+        PATH_DIRECTIVE,
+        WORKSPACE_DIRECTIVE,
+        CHECKIN_DIRECTIVE,
+        AUTOCHECKOUT_DIRECTIVE,
+        IGNORE_CONTENT_READERS_DIRECTIVE,
+        MAVEN_MOUNT_DIRECTIVE
```

Comment:
i do not understand why "maven:mount" is a directive. "mount" is a goal in sling-maven-plugin, what has it to do with the directives for the content loader?

Reply:
I thought this code is from you @stefanseifert : https://github.com/apache/sling-maven-plugin/blame/d67d5c1900782c68c1bfcbdc499c02cf31224916/sling-maven-plugin/src/main/java/org/apache/sling/maven/bundlesupport/fsresource/SlingInitialContentMounter.java#L97

Reply:
ah, you're right - this is a flag to instruct the maven plugin to ignore a certain directive when the "mount" goal is used. i will update the javadoc here a bit to give it a bit more explanation.